discover/portal/articles/130-forms/04-form-rules/05-autofill-rules.markdown

```diff
@@ -18,7 +18,7 @@ To follow this example:
 
     Make sure the output parameter is set to `nameCurentValue`. If you're unsure
```

Comment:
Please see my comment about the syntax of the _Parameter_ field of the _Outputs_ fields in the Data Provider article.